docs/source/reference/metrics.rst

```diff
@@ -0,0 +1,87 @@
+===========
+Metrics API
+===========
+
+BentoML provides metrics API via `Prometheus <https://prometheus.io/>`_. Prometheus
+provides a `Python Client <https://github.com/prometheus/client_python>`_ for
+instrumenting with Prometheus.
+
+BentoML provides a ``bentoml.metrics`` API as a drop-in replacement for ``prometheus_client``:
+
+.. code-block:: diff
+
+   diff --git a/service.py b/service.py
+   index acd8467e..0f3e6e77 100644
+   --- a/service.py
+   +++ b/service.py
+   @@ -1,11 +1,10 @@
+   -from prometheus_client import Summary
+   +from bentoml.metrics import Summary
+    import random
+    import time
+
+   REQUEST_TIME = Summary("request_processing_seconds", "Time spent processing request")
+
+   @REQUEST_TIME.time()
+   def process_request(t):
+       """A function that takes some time."""
+
+Difference between ``bentoml.metrics`` and ``prometheus_client``
+----------------------------------------------------------------
```

Comment:
I think this section should be moved above the line and examples above or just removed

```suggestion
```